src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val deleteBackupLocator = By.xpath(
+        "//*[@id='tbl_services']" +
+            "//*[contains(text(), 'com.atlassian.jira.service.services.export.ExportService')]" +
+            "/ancestor::tr" +
+            "//*[contains(@class, 'operations-list')]" +
+            "//*[contains(text(), 'Delete')]"
+    )
+
+    fun delete(): BackupConfiguration {
```

Comment:
To differentiate from `deleteBackupService`, we can call this: `deleteAllBackupServices`